client/src/components/FrienderPanel.jsx

```diff
@@ -19,8 +19,10 @@ class FrienderPanel extends React.Component {
       axios.post('/api/friends', {
         type: 'request',
         user: this.state.requestEmail
-      });
-      this.setState({requestEmail: ''});
+      })
```

Comment:
Should reset the requestEmail field immediately because in the future an MUI Snackbar will display any server-side errors